### PR TITLE
159/job description logo

### PR DESCRIPTION
**Pull request by razlze**
#### What is the purpose of your change?
- Adding the company logos to the Internship Card component

#### What areas of the application does this change impact? (changes API, folder structure, etc)
- Added a folder in assets to contain all company logos

#### What features/fixes are deferred/not handled in this PR?
- 

## Checklist
- [ ] Adds unit tests
- [ ] Has a work item linked to the PR

### CODE REVIEW

File: client/src/components/InternshipCompanyInfo.jsx
Comment (JennaXiao3) [inline client/src/components/InternshipCompanyInfo.jsx:33]: these brackets seem to be unnecessary, we can remove them

File: client/src/components/InternshipCompanyInfo.jsx
Comment (JennaXiao3) [inline client/src/components/InternshipCompanyInfo.jsx:14]: can we make this case insensitive in case there is a misspelling? ideally, we have enums for companies in the future => will also depend on the backend

File: client/src/components/InternshipCompanyInfo.jsx
Comment (JennaXiao3) [inline client/src/components/InternshipCompanyInfo.jsx:47]: nit: any reason for the extra space?

**Review by JennaXiao3 — CHANGES_REQUESTED** [review]

File: client/src/components/InternshipCompanyInfo.jsx
Comment (JennaXiao3) [reply client/src/components/InternshipCompanyInfo.jsx:33]: ^ the outer empty brackets

File: client/src/components/InternshipCompanyInfo.jsx
Comment (JennaXiao3) [inline client/src/components/InternshipCompanyInfo.jsx:45]: Might be better to use rem instead of px => also note that this size will change since we made the job details text smaller (maybe wait for that to be merged first)

File: client/src/components/InternshipCompanyInfo.jsx
Comment (JennaXiao3) [inline client/src/components/InternshipCompanyInfo.jsx:41]: nit: imo I think it might be better to use `<Avatar />` rather than `<Box />` because it has quite a few attributes such as `variant="rounded"`, `src`, and `alt` that are more specific to images (it is also easier for those who are reading your code to understand that this is an image rather than a regular box)

**Review by JennaXiao3 — CHANGES_REQUESTED** [review]

**Comment by JennaXiao3** [comment]
Let two small comments but otherwise, looks good to me!

File: client/src/components/InternshipCompanyInfo.jsx
Comment (razlze) [reply client/src/components/InternshipCompanyInfo.jsx:45]: Ok, i'll wait for that to be merged first. The 69 px was CSS from before so I just left it as is

Comment (JennaXiao3) [reply client/src/components/InternshipCompanyInfo.jsx]: it has been merged :) but also it looks fine even after merge so this should be good to merge 👍

**Review by JennaXiao3 — APPROVED** [review]

**Review by GaroTawidian — APPROVED** [review]

**Pull request merged**
